{
  "build": "py38haa95532_0",
  "build_number": 0,
  "channel": "https://repo.anaconda.com/pkgs/main/win-64",
  "constrains": [],
  "depends": [
    "python >=3.8,<3.9.0a0"
  ],
  "extracted_package_dir": "E:\\Courses\\Anaconda\\pkgs\\exceptiongroup-1.2.0-py38haa95532_0",
  "files": [
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/INSTALLER",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/LICENSE",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/METADATA",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/RECORD",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/REQUESTED",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/WHEEL",
    "Lib/site-packages/exceptiongroup-1.2.0.dist-info/direct_url.json",
    "Lib/site-packages/exceptiongroup/__init__.py",
    "Lib/site-packages/exceptiongroup/__pycache__/__init__.cpython-38.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_catch.cpython-38.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-38.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_formatting.cpython-38.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_suppress.cpython-38.pyc",
    "Lib/site-packages/exceptiongroup/__pycache__/_version.cpython-38.pyc",
    "Lib/site-packages/exceptiongroup/_catch.py",
    "Lib/site-packages/exceptiongroup/_exceptions.py",
    "Lib/site-packages/exceptiongroup/_formatting.py",
    "Lib/site-packages/exceptiongroup/_suppress.py",
    "Lib/site-packages/exceptiongroup/_version.py",
    "Lib/site-packages/exceptiongroup/py.typed",
    ".nonadmin"
  ],
  "fn": "exceptiongroup-1.2.0-py38haa95532_0.conda",
  "license": "MIT",
  "link": {
    "source": "E:\\Courses\\Anaconda\\pkgs\\exceptiongroup-1.2.0-py38haa95532_0",
    "type": 1
  },
  "md5": "be452f26d2d72763df792ee41b941637",
  "name": "exceptiongroup",
  "package_tarball_full_path": "E:\\Courses\\Anaconda\\pkgs\\exceptiongroup-1.2.0-py38haa95532_0.conda",
  "paths_data": {
    "paths": [
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/INSTALLER",
        "path_type": "hardlink",
        "sha256": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "sha256_in_prefix": "d0edee15f91b406f3f99726e44eb990be6e34fd0345b52b910c568e0eef6a2a8",
        "size_in_bytes": 5
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/LICENSE",
        "path_type": "hardlink",
        "sha256": "6e5070d765031e0ad403a1cbf90ae6d19a0228f802e320b7acff460aa72ed47c",
        "sha256_in_prefix": "6e5070d765031e0ad403a1cbf90ae6d19a0228f802e320b7acff460aa72ed47c",
        "size_in_bytes": 3704
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/METADATA",
        "path_type": "hardlink",
        "sha256": "1d998cf5f14e54cf261360d854a049606682f73813b6a22d1b1d7bd51f17ba2e",
        "sha256_in_prefix": "1d998cf5f14e54cf261360d854a049606682f73813b6a22d1b1d7bd51f17ba2e",
        "size_in_bytes": 6603
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/RECORD",
        "path_type": "hardlink",
        "sha256": "2b924eaf9842ae7889ece97d1aeeaa1b91842951f178768b1443794d32e4e712",
        "sha256_in_prefix": "2b924eaf9842ae7889ece97d1aeeaa1b91842951f178768b1443794d32e4e712",
        "size_in_bytes": 1535
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/REQUESTED",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/WHEEL",
        "path_type": "hardlink",
        "sha256": "1196c6921ec87b83e865f450f08d19b8ff5592537f4ef719e83484e546abe33e",
        "sha256_in_prefix": "1196c6921ec87b83e865f450f08d19b8ff5592537f4ef719e83484e546abe33e",
        "size_in_bytes": 81
      },
      {
        "_path": "Lib/site-packages/exceptiongroup-1.2.0.dist-info/direct_url.json",
        "path_type": "hardlink",
        "sha256": "a049c9b1baeaba827a6678f6686509f3c0d7a3ddb3cd462ca5146b0c83929755",
        "sha256_in_prefix": "a049c9b1baeaba827a6678f6686509f3c0d7a3ddb3cd462ca5146b0c83929755",
        "size_in_bytes": 94
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__init__.py",
        "path_type": "hardlink",
        "sha256": "ec31d2d210e4f9122cdc841cdd26d9541d3ed4c86248227d5e0bc4c849682fb3",
        "sha256_in_prefix": "ec31d2d210e4f9122cdc841cdd26d9541d3ed4c86248227d5e0bc4c849682fb3",
        "size_in_bytes": 1049
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/__init__.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "cf4c6be8c648bd11877a797e715b034efbc3b413f552931951f616fb64850b86",
        "sha256_in_prefix": "cf4c6be8c648bd11877a797e715b034efbc3b413f552931951f616fb64850b86",
        "size_in_bytes": 855
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_catch.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "406e553d624bbd0d0cd9472f6798088b4cf483faa25dfefe1f3dace148f33ad2",
        "sha256_in_prefix": "406e553d624bbd0d0cd9472f6798088b4cf483faa25dfefe1f3dace148f33ad2",
        "size_in_bytes": 3182
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "fb8edf023f61c6d2bb444d737e35e4317ed6d980d4e37ff61713cfc5ca405df4",
        "sha256_in_prefix": "fb8edf023f61c6d2bb444d737e35e4317ed6d980d4e37ff61713cfc5ca405df4",
        "size_in_bytes": 9294
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_formatting.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "adc66fbaf2804f180279005f16159e4047afe4afacf2f346caec90676dce8a8a",
        "sha256_in_prefix": "adc66fbaf2804f180279005f16159e4047afe4afacf2f346caec90676dce8a8a",
        "size_in_bytes": 11904
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_suppress.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "e43d9646a812967d582131b3f774c62cddbad52fba23a4cafa01cc2500073086",
        "sha256_in_prefix": "e43d9646a812967d582131b3f774c62cddbad52fba23a4cafa01cc2500073086",
        "size_in_bytes": 1073
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/__pycache__/_version.cpython-38.pyc",
        "path_type": "hardlink",
        "sha256": "243f19211e45c61619713aea9cc02067ee7727530287ecbc70fbef980a27316f",
        "sha256_in_prefix": "243f19211e45c61619713aea9cc02067ee7727530287ecbc70fbef980a27316f",
        "size_in_bytes": 230
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_catch.py",
        "path_type": "hardlink",
        "sha256": "926c74d4b904648e7ea66c61f404c43b8bf3a80b649fb9e357ee59f85d0aa7e2",
        "sha256_in_prefix": "926c74d4b904648e7ea66c61f404c43b8bf3a80b649fb9e357ee59f85d0aa7e2",
        "size_in_bytes": 4669
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_exceptions.py",
        "path_type": "hardlink",
        "sha256": "7fe58e5c7c3e9555800198c663c73e285c687c2bc37049785de3037aa7203328",
        "sha256_in_prefix": "7fe58e5c7c3e9555800198c663c73e285c687c2bc37049785de3037aa7203328",
        "size_in_bytes": 11010
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_formatting.py",
        "path_type": "hardlink",
        "sha256": "9ae98881193ad6d86fef03fa5b30a954db063e16191967df25e1b34278052f4b",
        "sha256_in_prefix": "9ae98881193ad6d86fef03fa5b30a954db063e16191967df25e1b34278052f4b",
        "size_in_bytes": 21005
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_suppress.py",
        "path_type": "hardlink",
        "sha256": "f3298ac7900570ed79178ecd7e7448f86f7c9a7e19057bb2b78aa22dde730f74",
        "sha256_in_prefix": "f3298ac7900570ed79178ecd7e7448f86f7c9a7e19057bb2b78aa22dde730f74",
        "size_in_bytes": 1338
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/_version.py",
        "path_type": "hardlink",
        "sha256": "3d96f686ea42f34d7ccf213690b58eddd919a14ad655174f06325d55f04d7547",
        "sha256_in_prefix": "3d96f686ea42f34d7ccf213690b58eddd919a14ad655174f06325d55f04d7547",
        "size_in_bytes": 164
      },
      {
        "_path": "Lib/site-packages/exceptiongroup/py.typed",
        "path_type": "hardlink",
        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "sha256_in_prefix": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
        "size_in_bytes": 0
      }
    ],
    "paths_version": 1
  },
  "requested_spec": "None",
  "sha256": "201ac8eb227e30d7f64f88f43f78570463171aec39c343a99818021dca5c9fb3",
  "size": 30992,
  "subdir": "win-64",
  "timestamp": 1706031611000,
  "url": "https://repo.anaconda.com/pkgs/main/win-64/exceptiongroup-1.2.0-py38haa95532_0.conda",
  "version": "1.2.0"
}